source/views/dictionary/report/editor.tsx

```diff
@@ -10,80 +10,66 @@ import {
 import {submitReport} from './submit'
 import type {WordType} from '../types'
 import type {TopLevelViewPropsType} from '../../types'
+import {NativeStackNavigationOptions} from '@react-navigation/native-stack'
+import noop from 'lodash/noop'
 
 type Props = TopLevelViewPropsType & {
```

Comment:
```suggestion
type Props = {
```

Reply:
To expand on this: there is no need to extend or try and mix the `TopLevelViewPropsType` into this type. Anything that satisfies the contract of `{route: {params: {item: WordType}}}` would fit here.

There is another pattern we could follow with interfaces, but this is essentially equivalent.

Reply:
Yeah! This is leftover from the non-hook version of navigation and I missed removing it from this file.